Index: build/libpod/container_internal_common.go
===================================================================
--- build.orig/libpod/container_internal_common.go
+++ build/libpod/container_internal_common.go
@@ -187,7 +187,7 @@ func (c *Container) generateSpec(ctx con
 	// Apply AppArmor checks and load the default profile if needed.
 	if len(c.config.Spec.Process.ApparmorProfile) > 0 {
 		updatedProfile, err := apparmor.CheckProfileAndLoadDefault(c.config.Spec.Process.ApparmorProfile)
-		if err != nil {
+		if err != nil || os.Getenv("AA_RELOAD") == "0" {
 			return nil, err
 		}
 		g.SetProcessApparmorProfile(updatedProfile)

